ci-operator/step-registry/ipi/conf/aws/byo-ipv4-pool-public/ipi-conf-aws-byo-ipv4-pool-public-commands.sh

```diff
@@ -13,12 +13,17 @@ CONFIG="${SHARED_DIR}/install-config.yaml"
 
 # Total Public IPs:
 ## bootstrap + (( Public LBs[API+Router] + Nat GWs) * Zones) = 1 + (3*2) = 7
-#zone_count=$(yq-go r -j "$CONFIG" | jq -r '.controlPlane.platform.aws.zones | length')
-#expected_ip_available=$(( ( zone_count * 3 ) + 1 ))
-
-##> TODO> setting fixed number of available IPs to have a buffer while a 'pool semaphore'
-# is not created.
-expected_ip_available=30
+zone_count=$(yq-go r -j "$CONFIG" | jq -r '.controlPlane.platform.aws.zones | length')
+expected_ip_available=$(( ( zone_count * 3 ) + 1 ))
```

Comment:
Currently the Router isn't supported and not allocating EIPs from custom pools, only resources created by installer.
So I believe if we want to save IPs, we can change the expression to something like this:

```suggestion
expected_ip_available=$(( ( zone_count * 2) + 1 ))
```